app.js

```diff
@@ -83,45 +77,103 @@ var img1 = document.getElementsByTagName('img')[0];
 var img3 = document.getElementsByTagName('img')[2];
 // what are we listening for? click
+
+// function getVotes(product) {
+//   if (totalclicks < 25){
+//     totalclicks++;
+//     product.votes++;
+//     displayThreeNewproducts();
+//   }
+//   else{
+//     //localstorage goes here
+//     displayVotes();
+//   }
+// }
+
+
 img1.addEventListener('click', function() {
   // add to votes for that Products
-  totalclicks++;
   if (totalclicks < 25){
+    totalclicks++;
     products1.votes++;
     displayThreeNewproducts();
   }
   else{
-    displayVotes();}
+    //localstorage goes here
+    displayVotes();
+  }
   // products1.votes = products1.votes + 1;
 
 });
 
 img2.addEventListener('click', function() {
   // what should we do?
   // add to votes for that Products
-  totalclicks++;
   if (totalclicks < 25){
+    totalclicks++;
     products2.votes++;
-
     displayThreeNewproducts();
   }
-  else{displayVotes();}
-  // products1.votes = products1.votes + 1;
-
+  else{
+    //localstorage goes here
+    displayVotes();
+  }
 });
 
 img3.addEventListener('click', function() {
   // what should we do?
   // add to votes for that Products
-  totalclicks++;
   if (totalclicks < 25){
+    totalclicks++;
     products3.votes++;
     displayThreeNewproducts();
   }
-  else{displayVotes();}
+  else{
+    //localstorage goes here
+    displayVotes();
+  }
+
   // products1.votes = products1.votes + 1;
 });
 
+var productData = JSON.parse(localStorage.getItem('products'));
+
+
+if(productData){
+  console.log('Storage Exists');
+  for(var i = 0; i < productData.length; i++){
+    new Products(productData[i].displayName, productData[i].filename, productData[i].votes, productData[i].timesDisplayed);
+  }
+  var products1 = Products.allproducts[0];
+  var products2 = Products.allproducts[1];
+  var products3 = Products.allproducts[2];
+}else{
+  products1 = new Products('Bag', 'img/bag.jpg');
+  products2 = new Products('Banana', 'img/banana.jpg');
+  products3 = new Products('Bathroom', 'img/bathroom.jpg');
+  new Products('Boots', 'img/boots.jpg');
+  new Products('Breakfast', 'img/breakfast.jpg');
+  new Products('Bubblegum', 'img/bubblegum.jpg');
+  new Products('Chair', 'img/chair.jpg');
+  new Products('Demon', 'img/cthulhu.jpg');
+  new Products('Duck Mask', 'img/dog-duck.jpg');
+  new Products('Dragon Meat', 'img/dragon.jpg');
+  new Products('Pen Utensils', 'img/pen.jpg');
+  new Products('Sweep Shoe', 'img/pet-sweep.jpg');
+  new Products('Scissors', 'img/scissors.jpg');
+  new Products('Sharks', 'img/shark.jpg');
+  new Products('Sweep', 'img/sweep.png');
+  new Products('Tauntaun', 'img/tauntaun.jpg');
+  new Products('Unicorn', 'img/unicorn.jpg');
+  new Products('Usb', 'img/usb.gif');
+  new Products('Water Can', 'img/water-can.jpg');
+  new Products('Wine Glass', 'img/wine-glass.jpg');
+
+  console.log('Storage does not exist');
+}
+
+
+
 
 function displayVotes(){
   var stopEvent = (function() {

```

Comment:
This 'IFFE'  and 'var' that you have here are actually not needed. You can simply call ```document.getElementById('stop').setAttribute('style', 'display:none');```. 
There is no need for the function.